### PR TITLE
Refine help text in cli

### PR DESCRIPTION
**Pull request by even-wei**
Signed-off-by: Even Wei <evenwei@infuseai.io>

** PR checklist **

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**
Enhancement

**What this PR does / why we need it**:
Refine help text in cli

**Which issue(s) this PR fixes**:
sc-26683

### CODE REVIEW

File: piperider_cli/cli.py [hunk 1]
```diff
@@ -145,19 +153,23 @@ def run(**kwargs):
...
     input = kwargs.get('input')
     workspace.generate_report(input=input)
 
 
-@cli.command(short_help='Compare two existing reports')
-@click.option('--base', default=None, type=click.Path(exists=True), help='Path of the base json report file')
-@click.option('--input', default=None, type=click.Path(exists=True), help='Path of the json report file to compare')
+@cli.command(short_help='Compare two existing reports.')
+@click.option('--base', default=None, type=click.Path(exists=True), help='Specify the base report file.')
+@click.option('--input', default=None, type=click.Path(exists=True), help='Specift the report file to be compared.')
```
Comment (jonycfu) [inline piperider_cli/cli.py:168]: `help='Specift ...` typo (Specify)

File: piperider_cli/cli.py [hunk 2]
```diff
@@ -106,9 +110,11 @@ def init(**kwargs):
         console.print(config)
 
 
-@cli.command(short_help='Test Configuration')
+@cli.command(short_help='Check project configuration.')
 @add_options(debug_option)
 def diagnose(**kwargs):
+    'Check project configuration, datasource, connections, assertion config and dbt manifest.'
```
Comment (jonycfu) [inline piperider_cli/cli.py:116]: stick to `configuration` or `config` for consistency
for both project and assertion

**Review by kentwelcome — APPROVED** [review]
LGTM

File: piperider_cli/cli.py [hunk 3]
```diff
@@ -145,19 +153,23 @@ def run(**kwargs):
         workspace.generate_report()
 
 
-@cli.command(short_help='Show report')
-@click.option('--input', default=None, type=click.Path(exists=True), help='Path of json report file')
+@cli.command(short_help='Generate a report.')
+@click.option('--input', default=None, type=click.Path(exists=True), help='Specify the raw result file.')
```
Comment (jonycfu) [inline piperider_cli/cli.py:157]: For new user, will they know what a raw result file is?
Maybe mention `.piperider/outputs`  (similar to how documentation site explains it)

File: piperider_cli/cli.py [hunk 4]
```diff
@@ -145,19 +153,23 @@ def run(**kwargs):
...
 @add_options(debug_option)
 def generate_report(**kwargs):
+    'Generate a report from the latest raw result or specified result.'
```
Comment (jonycfu) [inline piperider_cli/cli.py:160]: Same issue here with "raw result"

**Review by jonycfu — COMMENTED** [review]
Just some minor fixes ;)

File: piperider_cli/cli.py (hunk 1 above)
Comment (even-wei) [reply piperider_cli/cli.py]: fixed

File: piperider_cli/cli.py (hunk 2 above)
Comment (even-wei) [reply piperider_cli/cli.py]: fixed, use configuration.

File: piperider_cli/cli.py [hunk 5]
```diff
@@ -134,7 +134,7 @@ def diagnose(**kwargs):
 @click.option('--skip-dbt', is_flag=True, help='Skip running dbt.')
 @add_options(debug_option)
 def run(**kwargs):
-    'Profile data source, run assertions, and generate report(s).'
+    'Profile data source, run assertions, and generate report(s). By default, the raw results are saved in ".piperider/outputs" and the reports are saved in ".piperider/reports".'
```
Comment (jonycfu) [inline piperider_cli/cli.py:137]: Is `.piperider/reports` a new folder? I haven't seen it yet 
image

Comment (jonycfu) [reply piperider_cli/cli.py:137]: image
^ from generating reports

**Review by jonycfu — APPROVED** [review]

File: piperider_cli/cli.py (hunk 5 above)
Comment (even-wei) [reply piperider_cli/cli.py:137]: uh, reports were saved in '.piperider/reports'.
let me check it's correct or a bug.

Comment (DaveFlynn) [reply piperider_cli/cli.py:137]: Going forward reports are exported to .piperider/outputs, we have updated the Next docs accordingly. The live doc site is still for ver 0.2.0

Comment (even-wei) [reply piperider_cli/cli.py]: Fixed

**Pull request merged**